eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/utils/spanned/EmptyLinesTrimming.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.utils.spanned;
+
+import android.text.SpannableStringBuilder;
+import android.text.Spanned;
+
+import com.schedjoules.eventdiscovery.framework.utils.factory.Factory;
+
+
+/**
+ * {@link Spanned} decorator that trims leading and trailing empty lines.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class EmptyLinesTrimming extends AbstractSpanned
```

Comment:
Just realized that server won't trim the content of the html I suppose, so we need to do it on client side. So I will do this `Trimmed` `Spanned` then.